Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG alpine_version=3.13
+ARG alpine_version=3.13.6
```

Comment:
We are bumping the other images to 3.14.2

Reply:
Done